utils_test.go

```diff
@@ -324,14 +324,16 @@ func TestGetSubidLimits(t *testing.T) {
 
 	// fake login.defs data
 	fileData := `# some comments
+# a comment with SUB_UID_MIN and SUB_UID_MAX; should be ignored by parser
```

Comment:
Where/how are you handling/skipping comments?

Reply:
It's done by the combination of stripping whitespace in each scanned line, and then checking that the line starts with the tokens we are looking for (SUB_UID_MIN, etc.) If a line starts with anything else it's skipped.